app/views/mailing_list/steps/completed.html.erb

```diff
@@ -2,78 +2,35 @@
 
 <div data-sub-channel-id="<%= session.dig("mailinglist", "sub_channel_id") %>"></div>
 
-<% if show_welcome_guide? %>
-  <%= render(Registration::MailingListSignupConfirmationComponent.new(mailing_list_session)) %>
-<% else %>
-  <section>
-    <h1 class="heading-l heading--box-green">You've signed up</h1>
-
-    <p>
-      You'll receive a welcome email shortly. If your circumstances change, you can let us know at any time.
-    </p>
-
-    <h2 class="heading-m">Want to talk to someone?</h2>
-
-    <p>Our agents are experts who can answer questions about qualifications, routes into teaching or your application.</p>
-
-    <div class="mailing-list__actions">
-      <% if callback_available? %>
-        <%= link_to("Book a callback", callbacks_steps_path, class: "button") %>
-        <div>
-          <small>or call us now:</small>
-          <p class="telephone">0800 389 2500</p>
-        </div>
-      <% else %>
-        <div>
-          <small>Call us now:</small>
-          <p class="telephone">0800 389 2500</p>
-        </div>
-      <% end %>
-    </div>
-
-    <p>Monday to Friday between 8:30am and 5:30pm, except on <a href="https://www.gov.uk/bank-holidays">bank holidays</a>.</p>
-
-  </section>
-
-  <div class="blocks">
-    <%= render Content::GenericBlockComponent.new(
-      title: "Life as a teacher",
-      icon_image: "icon-school-black.svg",
-      icon_size: "40x30",
-      classes: "blocks__directory"
-    ) do %>
-      <ul>
-        <li><a href="/blog">Read teachers' stories</a></li>
-        <li><a href="https://schoolexperience.education.gov.uk/">Experience a real school</a></li>
-        <li><a href="/salaries-and-benefits">Teachers' salaries</a></li>
-      </ul>
-    <% end %>
-    <%= render Content::GenericBlockComponent.new(
-      title: "Your next steps",
-      icon_image: "icon-doc-black.svg",
-      icon_size: "40x35",
-      classes: "blocks__directory"
-    ) do %>
-      <ul>
-        <li><a href="/steps-to-become-a-teacher">How to become a teacher</a></li>
-        <li><a href="/train-to-be-a-teacher">Explore ways to train</a></li>
-        <li><a href="/funding-and-support">Find your funding options</a></li>
-      </ul>
-    <% end %>
-    <%= render Content::GenericBlockComponent.new(
-      title: "Speak to teachers",
-      icon_image: "icon-git-black.svg",
-      icon_size: "33x33",
-      classes: "blocks__directory"
-    ) do %>
-      <p class="block__text">
-        Attend an event and speak to current teachers or get your
-        questions answered by our expert advisers. You can
-        also find out what it's like to train and be in the
-        classroom.
-      </p>
-
-      <%= link_to "Find an event", events_path, class: "button button--white button--unpadded" %>
-    <% end %>
-  </div>
-<% end %>
+<section>
+  <h1 class="heading-l heading--box-green">You've signed up</h1>
```

Comment:
On the new adviser funnel, we say `<Name>, you're signed up` - do we want to replicate that on the mailing list to make it a bit more personal @philspray @ethax-ross ?